apps/condo/domains/contact/components/ContactsEditor/index.tsx

```diff
@@ -360,6 +362,88 @@ export const ContactsEditor: React.FC<IContactEditorProps> = (props) => {
         hideTabBar ? { display: 'none' } : {},
     [hideTabBar])
 
+    const residentInfo = useMemo(() => {
+        return (
+            <>
+                <Typography.Title level={4}>{ResidentMessage}</Typography.Title>
```

Comment:
Should we show this text even if we show tabs?

Reply:
you have a good eye, fixed it